client/packages/purchasing/src/purchase_order/ListView/ListView.tsx

```diff
@@ -93,6 +94,21 @@ const ListView: FC = () => {
         format: ColumnFormat.Date,
         accessor: ({ rowData }) => rowData.confirmedDatetime,
         sortable: true,
+        align: ColumnAlign.Left,
+      },
+      {
+        key: 'sentDatetime',
+        label: 'label.sent',
+        format: ColumnFormat.Date,
+        accessor: ({ rowData }) => rowData.sentDatetime,
+        align: ColumnAlign.Left,
+      },
+      {
+        key: 'requestedDeliveryDate',
+        label: 'label.requested-delivery-date',
+        format: ColumnFormat.Date,
+        accessor: ({ rowData }) => rowData.confirmedDatetime,
```

Comment:
another confirmed date 😉 
```suggestion
        accessor: ({ rowData }) => rowData.requestedDeliveryDate,
```

Reply:
Woops good catch

Reply:
Have applied! https://github.com/msupply-foundation/open-msupply/pull/9108/commits/e5e8403e5851337c2c3c976a5c85ddc6813e641c